language-plutus-core/src/Language/PlutusCore/Lexer/Type.hs

```diff
@@ -174,7 +172,4 @@ instance Pretty (Version a) where
     pretty (Version _ i j k) = pretty i <> "." <> pretty j <> "." <> pretty k
 
 allBuiltinNames :: [BuiltinName]
-allBuiltinNames = dropR 2 [minBound .. maxBound]  -- Ignoring the last two outdated 'BuiltinName's.
-
-dropR :: Int -> [a] -> [a]
-dropR n xs = take (length xs - n) xs
+allBuiltinNames = [minBound .. maxBound]
```

Comment:
This definition ensures that it's enough to add a new built-in to `BuiltinName` and it'll be handled by our tests automatically.